Integrate-CircleCI.md

```diff
@@ -0,0 +1,75 @@
+# CircleCI Integration
+This guide will demonstrate how to configure CircleCI and run your test script with Kobiton on CircleCI.
```

Comment:
remove "run your test script with Kobiton on CircleCI"
This article doesn't guide it